app/controllers/declaration/AdditionalFiscalReferencesController.scala

```diff
@@ -123,11 +122,8 @@ class AdditionalFiscalReferencesController @Inject()(
   private def updateCacheModels(itemId: String, updatedCache: Seq[AdditionalFiscalReference], redirect: Call)(
```

Comment:
My propose is to change the method name to use `model` not `models` or just inline this

In practice there is no point to extract this as a method to invoke method

Reply:
ok will do